RealmSwift-swift2.0/Tests/RealmCollectionTypeTests.swift

```diff
@@ -19,39 +19,65 @@
 import XCTest
 import RealmSwift
 
-class SwiftAggregateObjectList: Object {
-    let list = List<SwiftAggregateObject>()
+class CTTAggregateObject: Object {
```

Comment:
Initialism of CollectionTypeTests, to distinguish these types from similarly-named ones used elsewhere in the RealmSwift tests.